Automattic/jetpack
# wp-admin Site Default: default navigation to wp-admin when interface option is set to wp-admin

https://github.com/Automattic/dotcom-forge/issues/4155

See PT: pesfPa-lH-p2

Some users would like to be able to use the traditional wp-admin navigation. At the moment, this is possible for only certain pages and needs to be set per screen. There are also various screens added in Calypso that are not available in wp-admin but should still be available via the menu.

## Proposed changes:


This PR introduces a change where we read the `wpcom_admin_interface` option for a site. If this value exists, it can be either `wp-admin` or `calypso`. 

When the value is `wp-admin` some alterations have been made to keep the majority of menu items of the admin menu pointing to wp-admin pages (such as posts, pages, comments, media, etc.). At the same time, some items are still pointing to the calypso UI as is the case currently.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Go to or create a new Atomic site
- Install [Jetpack beta tester](https://jetpack.com/download-jetpack-beta/) plugin
- Via the Jetpack menu on your Atomic site, go to the Jetpack beta tester page, select Jetpack (manage)
- Search for the feature branch `add / atomic wpadmin menu` and activate that branch. 
- Your Atomic site should now be using that specific Jetpack plugin branch
- At this point, the menu should look like the regular Atomic site menu.
- Via `_cli` run `wp option update wpcom_admin_interface wp-admin` This will set the option that enables the default menu. 
- Check that the menu has changed and that the most menu items now point to wp-admin instead of Calypso.
